02_bubble/harirakr/main_test.go

```diff
@@ -0,0 +1,68 @@
+package main
+
+import (
+	"bytes"
+	"strconv"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+type Data struct {
+	name     string
+	input    []int
+	expected []int
+}
+
+type Sortfns func([]int) []int
+
+func TestMain(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+	main()
+	assert.Equal(t, "[1 2 3 5]\n", buf.String())
+}
+
+func TestSortFunctions(t *testing.T) {
```

Comment:
Changes look good to me. One test case can be added to cover negative number sorting also